specs/merge/beacon-chain.md

```diff
@@ -198,7 +200,9 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
     process_operations(state, block.body)
     # Pre-merge, skip execution payload processing
     if is_execution_enabled(state, block):
-        process_execution_payload(state, block.body.execution_payload, EXECUTION_ENGINE)  # [New in Merge]
+        # [New in Merge]
```

Comment:
The `if is_execution_enabled(state, block):` is also new in the Merge. Maybe unify the two comments?